src/ksyms.cpp

```diff
@@ -23,8 +42,66 @@ std::string Ksyms::resolve(uint64_t addr, bool show_offset)
   return symbol.str();
 }
 
-Ksyms::~Ksyms()
+#ifdef USE_BLAZESYM
+std::optional<std::string> Ksyms::resolve_blazesym_int(uint64_t addr,
+                                                       bool show_offset)
 {
-  if (ksyms_)
-    bcc_free_symcache(ksyms_, -1);
+  if (symbolizer_ == nullptr) {
+    symbolizer_ = blaze_symbolizer_new();
+    return std::nullopt;
```

Comment:
why do we want to return here?

Reply:
I think intention was to check `symbolizer_` for `NULL` again. Will fix. Thanks for catching.